android_beta/8.2beta.yml

```diff
@@ -0,0 +1,84 @@
+release:
+  release_date: '2024-11-18'
+  text: |
+    **These notes apply to Thunderbird for Android version 8.2 betas, first released November 18th, 2024**
+
+  groups:
+    - ' '
+
+  releases:
+    - version: '8.2b1'
+      release_date: '2024-11-18'
+
+notes:
+  ## Beta 1
+  # New
+  - note: Change account initials to use display name for better customizability
+    tag: new
+    group: 1
+
+  - note: The account icons no longer switch positions when selected
+    tag: new
+    group: 1
+
+  - note: Improve messaging for folder notification setup and clarify any missing requirements
+    tag: new
+    group: 1
+
+  # Changed
+  - note: Use the respective app name sent via the IMAP ID command
+    tag: changed
+    group: 1
+
+  - note: Change default funding cadence to monthly
+    tag: changed
+    group: 1
+
+  - note: Only enable unified inbox when there is more than one account
+    tag: changed
+    group: 1
+
+  - note: Update listing screenshots to remove the calendar icon from the mockup
+    tag: changed
+    group: 1
+
+  - note: Update listing icon sizes to be consistent between beta and release
+    tag: changed
+    group: 1
+
+
+  # Fixed
+  - note: >
+      Set the correct deletion policy for accounts imported via QR code (double check if yours
+      are set correctly!)
```

Comment:
We call this "delete policy" internally. However, the user-facing setting is called "When I delete a message" and can be found under *Settings → [Account] → Fetching mail*.